src/components/MapContainer/MapContainer.module.scss

```diff
@@ -18,15 +17,9 @@
       pointer-events: auto;
     }
 
-    > :nth-child(1) {
-      justify-self: start;
-    }
-
-    > :nth-child(2) {
+    > div {
       justify-self: center;
       display: flex;
-      align-items: center;
-      justify-content: center;
       gap: theme.$spacingXSmall;
     }
   }
```

Comment:
<img width="1679" height="363" alt="Screenshot 2025-08-28 at 15 23 11" src="https://github.com/user-attachments/assets/2bc207a0-a8d7-4fda-bea1-82266ae2c259" />

The Year and Region Select disappear due to change of this style, could you refactor?